src/containers/Tenant/Diagnostics/TopShards/TopShards.tsx

```diff
@@ -66,126 +69,119 @@ function stringToQuerySortOrder(value) {
     );
 }
 
-function dataTableToStringSortOrder(value = []) {
+function dataTableToStringSortOrder(value: SortOrder | SortOrder[] = []) {
+    if (!Array.isArray(value)) {
+        value = [value];
```

Comment:
Assignment to function parameter doesn't look great. And it causes warning from eslint